packages/components/src/ui/ariakit-tooltip/index.tsx

```diff
@@ -0,0 +1,98 @@
+/**
+ * External dependencies
+ */
+import { Tooltip, TooltipAnchor, useTooltipState } from 'ariakit/tooltip';
+
+/**
+ * WordPress dependencies
+ */
+import { cloneElement, isValidElement } from '@wordpress/element';
+import { useInstanceId } from '@wordpress/compose';
+import deprecated from '@wordpress/deprecated';
+
+/**
+ * Internal dependencies
+ */
+import { TOOLTIP_DELAY } from '../../tooltip/';
+import type { ToolTipProps } from './types';
+import Shortcut from '../../shortcut';
+import { positionToPlacement as __experimentalPopoverLegacyPositionToPlacement } from '../../popover/utils';
```

Comment:
Using the solution from Popover to add a deprecation warning to use placement over position while ensuring position still works - related to #44401